tests/test_querysets.py

```diff
@@ -220,22 +220,22 @@ def test_aggregate_no_grouping(self):
     def test_aggregate_with_filter(self):
         # When filter comes before aggregate
         qs = Person.objects_in(self.database).filter(first_name='Warren').aggregate(average_height='avg(height)', count='count()')
-        print qs.as_sql()
+        print(qs.as_sql())
         self.assertEquals(qs.count(), 1)
         for row in qs:
             self.assertAlmostEqual(row.average_height, 1.675, places=4)
             self.assertEquals(row.count, 2)
         # When filter comes after aggregate
         qs = Person.objects_in(self.database).aggregate(average_height='avg(height)', count='count()').filter(first_name='Warren')
-        print qs.as_sql()
+        print(qs.as_sql())
```

Comment:
Thanks for your help guys, and sorry for breaking python3 compatibility.

I have no objection to the proposed changes, the only things to consider are:
- Does changing everything to unicode strings cause any performance problems due to excessive conversion to/from utf-8?
- Does changing everything to unicode strings modify the API in any way (for example, are there methods that used to accept both byte strings and unicode strings and now don't?)

@ainich - it's a good idea to use continuous integration for testing both python2 and python3, my company uses Jenkins for this purpose so I'll add jobs for testing infi.clickhouse_orm there.